apps/twitch-bot/internal/command/helpers/helpers_test.go

```diff
@@ -0,0 +1,56 @@
+package helpers
+
+import (
+	"testing"
+)
+
+func TestMakeUniqueArray(t *testing.T) {
+	type TestCase struct {
+		description string
+		input       []string
+		expected    []string
+	}
+
+	testCases := []TestCase{
+		{
+			description: "with one element",
+			input:       []string{"abc"},
+			expected:    []string{"abc"},
+		},
+		{
+			description: "with the same elements",
+
+			input:    []string{"abc", "def", "abc"},
+			expected: []string{"abc", "def"},
+		},
+		{
+
+			description: "with the unique elements",
+			input:       []string{"abc", "def", "ghi"},
+			expected:    []string{"abc", "def", "ghi"},
+		},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.description, func(t *testing.T) {
+			uniqueArr := MakeUniqueArray(testCase.input)
+
+			if !elementsEqual(uniqueArr, testCase.expected) {
+				t.Errorf("Array elements are not equal: uniqueArray: %s, expected: %s", uniqueArr, testCase.expected)
+			}
+		})
+	}
+}
+
+func elementsEqual(arr []string, arr2 []string) bool {
```

Comment:
This is a great start my man! Really hyped to see tests are making their way into the sencha. 💯 

One improvement: Instead of creating your own little helpers for each test file, we can utilize https://github.com/stretchr/testify. I had used it at work before, and i really like it, and it comes with all the possible assertion helpers you need.

Let me know if you need help with figuring out how to use this.